src/style.css

```
@@ -1 +1,74 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+body,
+ul {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  background-color: #fff;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: stretch;
+}
+
+.logo {
+  padding: 10px 0;
+}
+
+.logoImage {
+  display: block; /*fixing white space below*/
```

```suggestion
  display: block; 
```
you can remove comments